specification/resources/apps/commit_app_rollback.yml

```
@@ -0,0 +1,34 @@
+operationId: commit_app_rollback
```

```suggestion
operationId: run_commit_app_rollback
```

`run` is a keyword that can prefix any operationID